README.rst

```diff
@@ -2,43 +2,117 @@
 Howitz - Zino web client with HTMx and Flask
 ============================================
 
+
+Running Howitz step-by-step overview
+====================================
+
+1. Preparation step:
+    1. Make sure you have ``python3``, ``pip`` and ``git`` installed.
+    2. Download Howitz from the `GitHub repo <https://github.com/Uninett/Howitz>`_.
+
+2. Installation step:
+    1. Create and activate `venv`, from the project root folder run::
+
+        $ python3 -m venv howitzvenv
+        $ source howitzvenv/bin/activate
+
+    2. Install dependencies, from the project root folder run::
+
+        $ pip install -e .
+
+    Read more about installation in the `Install safely`_ section.
+
+3. Configuration step:
+    The easiest way to configure Howitz is via a ``toml`` file.
+
+    1. Create an empty ``.howitz.toml`` file in the project root folder.
+    2. Copy the values from the example config file ``dev-howitz.toml`` to ``.howitz.toml``.
```

Comment:
This file includes a lot of setup of debug logging, which isn't normally necessary unless you are working as a developer on Howitz.  I found it quite confusing that running `flask -app howitz user list` outputs a bunch of DEBUG and INFO log statements before it actually lists my defined users in plaintext...

Reply:
I haven't found a good way to mute the logging that happens on startup unfortunately. Maybe we should ship a default logging config that does that, or use a flag to explicitly turn it on.